### ISSUE TITLE
Symbol market data "type" == spot but "spot" property == true and "future" property == true, so why the type is only "spot"?

### ISSUE THREAD

**Issue by dawadam** [ccxt/ccxt#16834]
### ؜

Hi,
On symbol market data.

The "type" property exists, his value is like "spot" or "swap", etc.

Why can a market data have a "spot" parameter at true, a future parameter at true, and type == "spot"?

I also noticed sometimes the "spot" parameter is true, the "margin" parameter is true, and the type == "spot".

Thank you to enlighten me.

**Comment by carlosmiei** [comment]
> Why can a market data have a "spot" parameter at true, a future parameter at true, and type == "spot"?

Hello @dawadam, That shouldn't happen, it might be a bug. Where are you seeing it?

**Comment by dawadam** [comment]
Ok, yet this is the case for many pairs.
And on several interchanges.

An example on Binance:

```
    const exchange = new ccxt.binance()
    await exchange.loadMarkets()

    const market = exchange.markets['ETH/BTC']
    console.log(market.spot) // true
    console.log(market.future) // true
    console.log(market.type) // 'spot'
```

**Comment by carlosmiei** [comment]
> Ok, yet this is the case for many pairs on Binance, exemple:
> 
> ```
>     const exchange = new ccxt.binance()
>     await exchange.loadMarkets()
> 
>     const market = exchange.markets['ETH/BTC']
>     console.log(market.spot) // true
>     console.log(market.future) // true
>     console.log(market.type) // 'spot'
> ```

@dawadam Yeah, that is not expected behavior, `market.future` should return false in this case. We will fix it soon, thanks for letting us know.

**Comment by ttodua** [comment]
all such issues were fixed.
let us know if anyone finds any issues.
